mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -168,4 +169,23 @@ bool TCPServerAcceptConnection(const base::PlatformFile server_socket,
 #endif  // defined(OS_NACL)
 }
 
+bool GetSelfIPAddress(std::string* address) {
+  struct hostent* host_entry;
+  char hostname[80];
+  gethostname(hostname, sizeof(hostname));
+  host_entry = gethostbyname(hostname);
```

Comment:
In my local machine, `gethostbyname` just returns 127.0.1.1.
I think `gethostbyname` is based on hosts file.
Hence, if there is no information about own IP address in hosts file, we cannot get own address.

Reply:
I think It's better to remove this code :)
I am going to update PR. Thank you.